### PR TITLE
Add live code exercises: design/color-contrast.mdx

### LINKED ISSUE #57: Add live code exercises: design/color-contrast.mdx
## What's the problem?
The tutorial's underlying framework, [exerslide](https://github.com/facebookarchive/exerslide), was archived by its owner and has not been updated for several years. We want to modernize by migrating the tutorial to [docusaurus](https://github.com/facebook/docusaurus/)

## How can you help?
Recreate the live code exercises for the [**Color Contrast** section](https://teachaccess.github.io/tutorial/#/14)

## How to do it?
Please see this [step-by-step guide](https://github.com/teachaccess/tutorial/blob/docusaurus/migration-instructions.md) for how to complete this task

Check out the Headings slide for an example ([before](https://github.com/teachaccess/tutorial/blob/master/slides/02-Developers/01-headings.html.md) and [after](https://github.com/teachaccess/tutorial/blob/docusaurus/docs/code/headings.mdx))

## Helpful links
- [Tutorial website](https://teachaccess.github.io/tutorial/#/14)
- [Exerslide file (old)](https://github.com/teachaccess/tutorial/blob/master/slides/03-Designers/01-color-contrast.html.md)
- [Docusaurus file (new)](https://github.com/teachaccess/tutorial/blob/docusaurus/docs/design/color-contrast.mdx)

### PR DESCRIPTION
**Pull request by RiyaJohn**
Fixes #57

### CODE REVIEW

**Comment by RiyaJohn** [comment]
@rachelruderman can you review this, I wasn't sure about rendering the images, because most browsers aren't allowed to access local file system for security reasons right.

File: docs/design/color-contrast.mdx [hunk 1]
```diff
@@ -3,6 +3,54 @@ title: Color contrast
...
+      <img
+        src='./images/contrast-better-4-6.png'
+        alt='house with better contrast' />
+    </div>
+    <span>Color Contrast -- 4.6: 1</span>
+  </div>
+  <div>
+    <div>
+      <img
+        src='./images/contrast-good-7-5.png'
```
Comment (rachelruderman) [inline docs/design/color-contrast.mdx:37]: Hi @RiyaJohn !! These images already exist in the `static` directory 🙂  to get them working, just update the filepath like so:

```suggestion
        src='/static/img/contrast-good-7-5.png'
```

**Comment by rachelruderman** [comment]
@RiyaJohn Thank you for your contribution!! Happy to merge once...
1. Image paths updated to use `static` directory
2. Images in `docs/design/images` are removed

Great job!

**Comment by rachelruderman** [comment]
@RiyaJohn FYI I just updated my suggestion -- I gave you the wrong image path 😛  (either way I'm happy to merge as is and update on my end -- I have quite a few image paths to update anyways)

**Review by rachelruderman — APPROVED** [review]
Thank you!!!

**Pull request merged**